src/app/components/shared/datetime/datetime/datetime.component.spec.ts

```diff
@@ -0,0 +1,230 @@
+    // we should therefore see the hours increase by 6
+    name: "should display the correct dateTime if the Luxon dateTime object is a different utc offset to the users local timezone",
+    // by using +02:00, we know the offset, but not the timezone
+    // however, because this value will be localized to the users local timezone, we should see the timezone emitted
+    // in the users local timezone
+    value: DateTime.fromISO("2020-01-01T12:10:11.000+02:00"),
+    expectedText: "2020-01-01 18:10:11",
+    // notice that the tooltips are still in the test runners mock timezone
+    // this is because the emitted date/time has been localized to the test runners timezone
+    expectedTooltip: "2020-01-01 18:10:11.000 Australia/Perth +08:00",
+    expectedDateTimeAttribute: "2020-01-01T18:10:11.000+08:00",
+  },
+  {
+    // this test is the same as the previous offset test, except that we know both the timezone and offset
+    // in this test. (as opposed to the previous test where we only knew the offset)
+    name: "should display the correct DateTime if the Luxon dateTime object is a different timezone to the users local timezone",
+    // by using Australia/Sydney, we know the timezone and offset
+    value: DateTime.fromISO("2020-01-01T12:10:11.000Z").setZone(
+      "Australia/Sydney",
+      { keepLocalTime: true }
+    ),
+    expectedText: "2020-01-01 09:10:11",
+    expectedTooltip: "2020-01-01 09:10:11.000 Australia/Perth +08:00",
+    expectedDateTimeAttribute: "2020-01-01T09:10:11.000+08:00",
+  },
+  {
+    name: "should increment date if timezone localization causes the date to increment",
+    value: DateTime.fromISO("2020-01-01T23:10:11.000Z"),
+    expectedText: "2020-01-02 07:10:11",
+    expectedTooltip: "2020-01-02 07:10:11.000 Australia/Perth +08:00",
+    expectedDateTimeAttribute: "2020-01-02T07:10:11.000+08:00",
+  },
+  {
+    name: "should only display the date if dateOnly is set",
+    value: DateTime.fromISO("2020-01-01T12:10:11.000Z"),
+    expectedText: "2020-01-01",
+    expectedTooltip: "2020-01-01 20:10:11.000 Australia/Perth +08:00",
+    dateOnly: true,
+    expectedDateTimeAttribute: "2020-01-01T20:10:11.000+08:00",
+  },
+  {
+    name: "should only display the time if timeOnly is set",
+    value: DateTime.fromISO("2020-01-01T12:10:11.000Z"),
+    expectedText: "20:10:11",
+    expectedTooltip: "2020-01-01 20:10:11.000 Australia/Perth +08:00",
+    timeOnly: true,
+    expectedDateTimeAttribute: "2020-01-01T20:10:11.000+08:00",
+  },
+  {
+    name: "should display full datetime if both dateOnly and timeOnly are set",
+    value: DateTime.fromISO("2020-01-01T12:10:11.000Z"),
+    expectedText: "2020-01-01 20:10:11",
+    expectedTooltip: "2020-01-01 20:10:11.000 Australia/Perth +08:00",
+    dateOnly: true,
+    timeOnly: true,
+    expectedDateTimeAttribute: "2020-01-01T20:10:11.000+08:00",
+  },
+  {
+    name: "should localize a Luxon DateTime object with a timezone offset",
```

Comment:
It's supposed to be different because it contains an offset

However, as stated in a following comment: By adding offsets to all tests, we remove the need for this test.

I have removed this test

Reply:
Good catch, removed